paper/paper.md

```diff
@@ -20,12 +20,12 @@ bibliography: paper.bib
 # Summary
 
 Wavelets are widely used in various disciplines to analyse signals both in space and scale.
-Whilst many fields measure data on manifolds (i.e. the sphere), often data are only observed on a partial region of the manifold.
-Wavelets are a typical approach to data of this form, but the wavelet coefficients which overlap with the boundary become contaminated and must be removed for accurate analysis.
+Whilst many fields measure data on manifolds (i.e., the sphere), often data are only observed on a partial region of the manifold.
```

Comment:
I hate the American way of doing it, but I will concede 😛

Reply:
I didn't suggest changing whilst... 🙂